### PR TITLE
Fix apache test for jobs with SYSTEM_ROLE=kvm

### PR DESCRIPTION
**Pull request by asdil12**
Ticket: https://progress.opensuse.org/issues/56747
Verification: http://artemis.suse.de/tests/1744

### CODE REVIEW

File: tests/console/apache.pm [hunk 1]
```diff
@@ -102,8 +102,9 @@ sub run {
     # We stop current webserver and prepare another http2-prefork environment
     systemctl 'stop apache2';
 
-    if (is_sle('12-SP2+')) {
+    if (is_sle('12-SP2+') && !check_var('SYSTEM_ROLE', 'kvm')) {
         # Create directory for the new instance and prepare config
+        zypper_call('se apache2', exitcode => [0, 104]);
```
Comment (SergioAtSUSE) [inline tests/console/apache.pm]: What is this for?

Comment (asdil12) [reply tests/console/apache.pm]: debug relict - removed

File: tests/console/apache.pm [hunk 2]
```diff
@@ -102,7 +102,7 @@ sub run {
     # We stop current webserver and prepare another http2-prefork environment
     systemctl 'stop apache2';
 
-    if (is_sle('12-SP2+')) {
+    if (is_sle('12-SP2+') && !check_var('SYSTEM_ROLE', 'kvm')) {
```
Comment (foursixnine) [inline tests/console/apache.pm]: I would add a record info mentioning that the http2-prefork tests are being skipped, 'cause we're basically disabling half of the test. Or even better, let's not disable it by adding/using the missing files to the data directory

Comment (asdil12) [reply tests/console/apache.pm]: We can't do that as apache get's installed from a different test module.
Also disabling this part of the test doesn't really decrease coverage as it still runs in the non-kvm scenario.

Comment (asdil12) [reply tests/console/apache.pm]: Also I don't want to mess with the system by changing some configuration.
That would IMHO decrease the overall coverage as we wouldn't be testing anymore what a customer would experience in the kvm role.

Comment (asdil12) [reply tests/console/apache.pm]: But I'll add the `record_info`

**Comment by SergioAtSUSE** [comment]
Waiting for @foursixnine

**Comment by asdil12** [comment]
@foursixnine had a look at the code in person.

**Pull request merged**